GridTariffAPI/Mdmx/MdmxClient.cs

```diff
@@ -31,22 +31,17 @@ public MdmxClient(IHttpClientFactory httpClientFactory, IMdmxAccessTokenService
         }
 
 
-        public async Task<List<MeteringPointMaxConsumption>> GetVolumeAggregationsForThisMonthAsync(List<string> meteringPointIds)
+        public async Task<List<MeteringPointMaxConsumption>> GetMaxConsumptionsAsync(List<string> meteringPointIds)
         {
-            var localNow = TimeZoneInfo.ConvertTimeFromUtc(DateTime.UtcNow, _config.TimeZoneForMonthLimiting);
-            var localMonthStart = new DateTime(localNow.Year, localNow.Month, 1, 0, 0, 0, localNow.Kind);
-
-            var request = new VolumeAggregationBatchQuery
+            var request = new MaxConsumptionAggregationBatchQuery
             {
                 MeteringPointIds = meteringPointIds.ToArray(),
-                Register = "ActivePlus",
-                MeasurementTimeGe = localMonthStart,
-                MeasurementTimeLe = localNow
+                MeasurementTimeLe = null // null for the current MaxConsumption
```

Comment:
Her skal det vel hentes verdier fra inneværende måned.
Hvordan håndteres dette, burde det ikke vært "MeasurementTimeGe" med her ?

Reply:
Den nye operasjonen serverer MaxConsumption, med den logikken som hører til. Så jeg har plassert ansvaret for å ta volumene fra månedsstart til angitt tidspunkt der. Og med MeasurementTimeLe er det fra månedsstart til nå. Høres det greit ut?

Reply:
Antok nesten det. Høres veldig bra ut.